midcli/command/override/interface.py

```diff
@@ -20,14 +20,15 @@
 @rows_processor
 def patch_interface(context, interfaces):
     for interface in interfaces:
-        interface["aliases"] = [
-            (
-                alias["address"]
-                if {"INET": 32, "INET6": 128}[alias["type"]] == alias["netmask"]
-                else f"{alias['address']}/{alias['netmask']}"
-            )
-            for alias in interface["aliases"]
-        ]
+        if "aliases" in interface:
```

Comment:
Forgive me as I'm just trying to understand this program a little more. When you pass `query a` what happens to the `a`? Does that get passed as a filter/option to the `query` method? Why does checking for `aliases` first work around this?

Also, you're overwriting the `aliases` key in `interfaces` with a list of tuples with a single string? I'm assuming this is for easier printing? Sorry, this list comprehension is quite incomprehensible in my humble opinion. Again, not asking for any changes just to trying to understand a little more than just the surface.